source/query/atlas-search.txt

```diff
@@ -12,11 +12,23 @@ Run Atlas Search Queries
    :depth: 1
    :class: singlecol
 
-You can run Atlas Search :pipeline:`$search` and :pipeline:`$searchMeta`
-queries from |compass| if you :ref:`created the Atlas Search index
-<compass-create-search-index>` for data on the Atlas cluster. You can
-also run :pipeline:`$vectorSearch` queries from |compass| if you have an
-existing Atlas Vector search index on your data in Atlas. 
+Atlas Search queries take the form of an :ref:`aggregation pipeline stage 
+<compass-agg-builder>`. If you have an existing Atlas Search Index for data on 
+your Atlas cluster, you can run :pipeline:`$search` and :pipeline:`$searchMeta` 
+queries. If you have an  existing Atlas Vector search index on your data in 
+Atlas, you can also run :pipeline:`$vectorSearch` queries. 
+
+Before You Begin 
+----------------
+
+Create an :ref:`Atlas Search index <compass-create-search-index>` or an 
+:ref:`Atlas Vector Search index <compass-create-vector-search>`.
+
+.. note:: 
+   
+   You can manage Atlas Search indexes in the |compass-short| UI if your 
+   deployment is local, has an Atlas cluster tier of M10 or larger, and runs 
```

Comment:
(minor suggestion)
"... has an Atlas M10 cluster tier or larger ..." 

(another minor suggestion)
This info might be more scannable in a list?